File: 03. Programming - C#/TestPreparation4Example/Circle.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace TestPreparation4Example
{
    class Circle:Shape
    {
        private double r;
        public double R
        {
            get
            {
                return r;
            }set => r = value;
        }
        public Circle(string name, int numberOfSides,string type, double r)
            : base(name, numberOfSides, type)
        {
            this.r = r;
        }
        public override double calculateArea()
        {
            return Math.PI * r * r;
        }

        public override double calculatePerimeter()
        {
            return 2 * Math.PI * r;
        }
    }
}

File: 02. Programming - Java/Lesson7/src/uni/fmi/bachelors/Circle.java
package uni.fmi.bachelors;

public class Circle extends Shape {
	
	private double r;

	public Circle(String type, int numberOfSides, double r) {
		super(type, numberOfSides);
		this.r=r;
	}

	@Override
	public double calculatePerimeter() {
		return 2*Math.PI*r;
	}

	@Override
	public double calculateArea() {
		return Math.PI*r;
	}

	public double getR() {
		return r;
	}

	public void setR(double r) {
		this.r = r;
	}

	
}
